services/release-hatch/deploy.sh

```diff
@@ -1,8 +1,25 @@
 #!/bin/bash
 set -euo pipefail
 
-# start the docker container
 DIR=~opensafely/release-hatch
+UP_ARGS=
+
+# check for cert updates
+echo "Checking for certificate updates"
+openssl s_client -showcerts -connect backends.opensafely.org:443 </dev/null 2>/dev/null| sed -n '/^-----BEGIN CERT/,/^-----END CERT/p' > $DIR/certs/fullchain.latest.pem
+# if cert has changed
+if ! diff -q $DIR/certs/fullchain.pem $DIR/certs/fullchain.latest.pem; then
+    echo "New certificates found, updating"
+    cp certs/fullchain.latest.pem certs/fullchain.pem
```

Comment:
```suggestion
    cp $DIR/certs/fullchain.latest.pem $DIR/certs/fullchain.pem
```

Reply:
Hmm, I tried to get rid of $DIR references, because we can now assume that paths are relative to the justfile location (which is I think I think is nice about just)

I agree we should be consistent, but I'm leaning towards dropping $DIR from both, wdyt?

Reply:
Ah!

I need to delete this file!

Reply:
All the work is now in the justfile, this was left over changes from my early attempts before I switched to moving it to just